tests/examples/connext_dds/request_reply/cs/PrimesRequester.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Sleep to try to avoid DIT-194
+# The replier run for 20 seconds, so we can sleep for 10 seconds
+# to try to avoid the issue
+sleep 10
+
+cd ${BaseExamplePath}/cs
+/usr/bin/dotnet run -- --requester $@
```

Comment:
Why use `/bin` or `/usr/bin` explicitly? Would it no be best to rely on whatever is on `PATH`?

Reply:
TBD toguether with future improvements